### ISSUE TITLE
firstValue lastValue

### ISSUE THREAD

**Issue by marqh** [opengeospatial/netcdf-ld#49]
```
description | The last value in the data payload of a 1 dimensional array.
domain | Array
label | lastValue


description | The first value in the data payload of a 1 dimensional array.
domain | Array
label | firstValue
```

**Comment by marqh** [comment]
coordFirstValue

coordLastValue

**Comment by marqh** [comment]
arrayFirstValue

arrayLastValue

**Comment by adamml** [comment]
Thinking about this a bit more, and running through the CF Conventions document too...

Is there a case where arrayFirstValue does not equal arrayMinimumValue and where arrayLastValue does not equal arrayMaximumValue for coordinates?I was thinking these could be (even) more generic predicates.

**Comment by marqh** [comment]
hi @adamml 

ordering is an issue, as CF allows coordinate variables to be ordered increasing or decreasing

netCDF is slightly more lax, I think, in it's definition in the netCDF User Guide

I think that I'd prefer to stick with a name that indicates that it's taken from the order of values within the file, rather than providing insight into whether this is the minimum value, as that takes more processing to understand, and is then less bound to the indexing operations.

I think the key value is enabling an index operation to know what it's doing, as this is what can actually be done on a file.

on this basis, the use of 'minimum' and 'maximum' carries some risk, in my view

**Comment by adamml** [comment]
Thanks @marqh. Happy to stick with arrayFirstValue, arrayLastValue in that case.

**Comment by marqh** [comment]
closed by #47
